apps/v1/types.go

```diff
@@ -292,18 +292,18 @@ type DeploymentTriggerImageChangeParams struct {
 type DeploymentConfigStatus struct {
 	// LatestVersion is used to determine whether the current deployment associated with a deployment
 	// config is out of sync.
-	LatestVersion int64 `json:"latestVersion" protobuf:"varint,1,opt,name=latestVersion"`
+	LatestVersion int64 `json:"latestVersion,omitempty" protobuf:"varint,1,opt,name=latestVersion"`
```

Comment:
These should not be omit empty.  This would be a breaking change to JSON users, which have always set these fields.

Reply:
@soltysh why not `// +optional` ?

Reply:
I think we need `// +optional` but at the status level not here, looking at other types. Working on fixes.